deploy/vagrant/scripts/tinkerbell.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# abort this script on errors
+set -euxo pipefail
+
+whoami
+
+cd /vagrant
+
+setup_docker() (
```

Comment:
What was the issue preventing us from using setup.sh via curl? I thought it added compose and docker as part of setup.sh?

Reply:
See https://github.com/tinkerbell/tink/pull/147